src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -155,12 +213,37 @@ describe('EntryFieldsComponent', () => {
   it('when a start hour is updated, then dispatch UpdateActiveEntry', () => {
     component.activeEntry = entry ;
     component.setDataToUpdate(entry);
+    const newHour = moment().format('HH:mm:ss');
+    component.entryForm.patchValue({ start_hour : newHour});
     spyOn(store, 'dispatch');
 
     component.onUpdateStartHour();
     expect(store.dispatch).toHaveBeenCalled();
   });
 
+  it('when a start hour is update, then select the last time entry', async(() => {
```

Comment:
Correct me if I am wrong, I think this is what you are testing here.

```
When start_time is updated, component.last_entry is equal to...
```

Reply:
Sure, another way would be:
`When start_time is updated, component.last_entry is equal to last time entry in the state`